codeflash/cli_cmds/cmd_init.py

```diff
@@ -578,8 +578,20 @@ def configure_pyproject_toml(setup_info: SetupInfo) -> None:
     formatter_cmds = []
     if formatter == "black":
         formatter_cmds.append("black $file")
+        print("checking if black exists")
+        try:
+            import black
```

Comment:
we should do this checking exactly the same way we run black when we do the formatting. I believe it runs as a subprocess. We should detect with that method if black is installed. The same goes for ruff formatter